rice-datanode/source/ClientDatanodeProtocolImpl.cc

```diff
@@ -142,21 +153,33 @@ void ClientDatanodeTranslator::InitServer() {
  * (See rpcserver.cc - it's where these will get called from)
  */
 void ClientDatanodeTranslator::RegisterClientRPCHandlers() {
-  using namespace std::placeholders; // for `_1`
-
-  // The reason for these binds is because it wants static functions, but we want to give it member functions
-  // http://stackoverflow.com/questions/14189440/c-class-member-callback-simple-examples
+  // The reason for these binds is because it wants static functions, but we
+  // want to give it member functions
 
   server.register_handler("getReplicaVisibleLength",
-                          std::bind(&ClientDatanodeTranslator::getReplicaVisibleLength, this, _1));
-  server.register_handler("refreshNamenodes", std::bind(&ClientDatanodeTranslator::refreshNamenodes, this, _1));
-  server.register_handler("deleteBlockPool", std::bind(&ClientDatanodeTranslator::deleteBlockPool, this, _1));
+                          std::bind(
+                            &ClientDatanodeTranslator::getReplicaVisibleLength,
+                            this, std::placeholders::_1));
```

Comment:
sticking to our convention this should be 
`using std::placeholders::_1;` above

Reply:
good catch